applications/tasks/src/kz_account_crawler.erl

```diff
@@ -210,239 +201,13 @@ check_then_process_account(AccountId, {'ok', AccountJObj}) ->
-    AccountJObj2 = kz_account:remove_low_balance_tstamp(AccountJObj1),
-    _ = kz_util:account_update(AccountJObj2),
-    'ok'.
-
--spec maybe_low_balance_notify(kz_account:doc(), kz_transaction:units()) -> 'ok'.
-maybe_low_balance_notify(AccountJObj, CurrentBalance) ->
-    case kz_account:low_balance_enabled_exists(AccountJObj) of
-        'false' ->
-            lager:debug("low balance notification enabled key not present, using deprecated check"),
-            maybe_low_balance_notify_deprecated(AccountJObj, CurrentBalance);
-        'true' ->
-            maybe_low_balance_notify(AccountJObj, CurrentBalance, kz_account:low_balance_enabled(AccountJObj))
-    end.
-
--spec maybe_low_balance_notify(kz_account:doc(), kz_transaction:units(), boolean()) -> 'ok'.
-maybe_low_balance_notify(_AccountJObj, _CurrentBalance, 'false') ->
-    lager:debug("low balance notification disabled");
-maybe_low_balance_notify(AccountJObj, CurrentBalance, 'true') ->
-    lager:debug("low balance notification enabled"),
-    case kz_account:low_balance_tstamp(AccountJObj) of
-        LowBalanceSent when is_number(LowBalanceSent) ->
-            Cycle = ?LOW_BALANCE_REPEAT,
-            Diff = kz_time:now_s() - LowBalanceSent,
-            case Diff >= Cycle of
-                'true' -> notify_of_low_balance(AccountJObj, CurrentBalance);
-                'false' ->
-                    lager:debug("low balance alert sent ~w seconds ago, repeats every ~w", [Diff, Cycle])
-            end;
-        _Else -> notify_of_low_balance(AccountJObj, CurrentBalance)
-    end.
-
--spec maybe_low_balance_notify_deprecated(kz_account:doc(), kz_transaction:units()) -> 'ok'.
-maybe_low_balance_notify_deprecated(AccountJObj, CurrentBalance) ->
-    case kz_account:low_balance_sent(AccountJObj) of
-        'true' -> lager:debug("low balance alert already sent");
-        'false' -> notify_of_low_balance(AccountJObj, CurrentBalance)
-    end.
-
--spec notify_of_low_balance(kz_account:doc(), kz_transaction:units()) -> 'ok'.
-notify_of_low_balance(AccountJObj, CurrentBalance) ->
-    AccountId = kz_account:id(AccountJObj),
-    lager:debug("sending low balance alert for account ~s with balance ~w"
-               ,[AccountId, CurrentBalance]),
-
-    Req = [{<<"Account-ID">>, AccountId}
-          ,{<<"Current-Balance">>, CurrentBalance}
-           | kz_api:default_headers(?APP_NAME, ?APP_VERSION)
-          ],
-    kapps_notify_publisher:cast(Req, fun kapi_notifications:publish_low_balance/1),
-
-    update_account_low_balance_sent(AccountJObj).
-
--spec update_account_low_balance_sent(kz_account:doc()) -> 'ok'.
-update_account_low_balance_sent(AccountJObj0) ->
-    AccountJObj1 = kz_account:set_low_balance_sent(AccountJObj0),
-    AccountJObj2 = kz_account:set_low_balance_tstamp(AccountJObj1),
-    _ = kz_util:account_update(AccountJObj2),
+    _ = tasks_bindings:map(<<"tasks.account_crawler">>, [AccountId, AccountJObj]),
```

Comment:
since the modules binding to account crawler shouldn't depend on the execution of others, you can add `pmap` to `tasks_bindings` and use it here. check kazoo_bindings `pmap`